airflow-core/src/airflow/ui/public/i18n/locales/pt/common.json

```
@@ -0,0 +1,343 @@
+{
+  "admin": {
+    "Config": "Configuração",
+    "Connections": "Conexões",
+    "Plugins": "Plugins",
+    "Pools": "Pools",
+    "Providers": "Providers",
+    "Variables": "Variáveis"
+  },
+  "allOperators": "Todos os Operadores",
+  "appearance": {
+    "appearance": "Aparência",
+    "darkMode": "Modo Escuro",
+    "lightMode": "Modo Claro",
+    "systemMode": "Seguir Configuração do Sistema"
+  },
+  "asset_many": "Assets",
+  "asset_one": "Asset",
+  "asset_other": "Assets",
+  "asset_zero": "Nenhum asset",
+  "assetEvent_many": "Eventos de Asset",
+  "assetEvent_one": "Evento de Asset",
+  "assetEvent_other": "Eventos de Asset",
+  "assetEvent_zero": "Nenhum evento de asset",
+  "backfill_many": "Backfills",
+  "backfill_one": "Backfill",
+  "backfill_other": "Backfills",
+  "backfill_zero": "Nenhum backfill",
+  "browse": {
+    "auditLog": "Log de Auditoria",
+    "requiredActions": "Ações Necessárias",
+    "xcoms": "XComs"
+  },
+  "collapseDetailsPanel": "Recolher Painel de Detalhes",
+  "createdAssetEvent_many": "Eventos de Asset Criados",
+  "createdAssetEvent_one": "Evento de Asset Criado",
+  "createdAssetEvent_other": "Eventos de Asset Criados",
+  "createdAssetEvent_zero": "Nenhum evento de asset criado",
+  "dag_many": "DAGs",
+  "dag_one": "DAG",
+  "dag_other": "DAGs",
+  "dag_zero": "Nenhum DAG",
+  "dagDetails": {
+    "catchup": "Catchup",
+    "dagRunTimeout": "Tempo Limite da Execução do DAG",
+    "defaultArgs": "Argumentos Padrão",
+    "description": "Descrição",
+    "documentation": "Documentação do DAG",
+    "fileLocation": "Local do Arquivo",
+    "hasTaskConcurrencyLimits": "Tem Limite de Concorrência de Tarefas",
+    "lastExpired": "Último Expirado",
+    "lastParsed": "Último Parseado",
+    "lastParseDuration": "Duração do Último Parse",
```

```suggestion
    "lastParseDuration": "Duração do Último Parse",
    "lastParsed": "Último Parseado",
```

(according to static checks)